index.html

```diff
@@ -8,14 +8,30 @@
-      <p>The Open Biological and Biomedical Ontology (OBO) Foundry is a collective of ontology developers that are committed to collaboration and adherence to shared principles. The mission of the OBO Foundry is to develop a family of interoperable ontologies that are both logically well-formed and scientifically accurate. To achieve this, OBO Foundry participants voluntarily adhere to and contribute to the development of an evolving set of principles including <a href="/principles/fp-001-open.html">open use</a>, <a href="/principles/fp-010-collaboration.html">collaborative development</a>, <a href="/principles/fp-005-delineated-content.html">non-overlapping and strictly-scoped content</a>, and common <a href="/principles/fp-002-format.html">syntax</a> and <a href="/principles/fp-007-relations.html">relations</a>, based on ontology models that work well, such as the Gene Ontology (GO).</p>
-      <p>The OBO Foundry is overseen by an Operations Committee with <a href="/docs/EditorialWG.html">Editorial</a>, <a href="/docs/TechnicalWG.html">Technical</a> and <a href="/docs/OutreachWG.html">Outreach</a> working groups. The processes of the Editorial working group are modelled on the journal refereeing process. A complete treatment of the OBO Foundry is given in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2814061/">&quot;The OBO Foundry: coordinated evolution of ontologies to support biomedical data integration&quot;</a>.</p>
-      <p>On this site you will find a table of ontologies, available in several formats, with details for each, and documentation on <a href="/principles/fp-000-summary.html">OBO Principles</a>.</p>
-      <p>You can contribute to this site using GitHub <a href="https://github.com/OBOFoundry/OBOFoundry.github.io">OBOFoundry/OBOFoundry.github.io</a> or get in touch with us by joining our mail list <a href="https://groups.google.com/forum/#!forum/obo-discuss">https://groups.google.com/forum/#!forum/obo-discuss</a>.</p>
+  <b>Learn about OBO best practices and community resources</b>
+<ul>
+<li> <a href="http://www.obofoundry.org/about">More about the OBO Foundry</a></li>
+<li> <a href="http://www.obofoundry.org/principles/fp-000-summary.html">OBO Foundry principles</a></li>
+<li> <a href="https://github.com/jamesaoverton/obo-tutorial">OBO tutorial</a></li>
+<li> <a href="http://www.obofoundry.org/resources">Ontology browsers, tutorials, and tools</a></li>
+</ul>
+
+<b>Participate</b>
+<ul>
+<li> <a href="https://groups.google.com/forum/#!forum/obo-discuss">Join the OBO mailing list</a></li>
+<li> <a href="http://obofoundry.org/docs/OperationsCommittee.html">OBO Foundry Operations and Working Groups</a></li>
+<li> <a href="https://github.com/OBOFoundry/OBOFoundry.github.io/issues">Submit bug reports or suggestions for improvement via GitHub</a></li>
+<li> <a href="http://www.obofoundry.org/faq/how-do-i-submit-my-ontology.html">Submit your ontology to be considered for inclusion in the OBO Foundry</a></li>
```

Comment:
might be how-do-i-register-my-ontology.html (that page name has been switched back, I think)

Reply:
yes, should be register

Reply:
I think this page needs to be reverted back to say register:
https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/faq/how-do-i-submit-my-ontology.md

(I thought I did this? But I guess not)

Reply:
i thought you did too, that's why i'm so confused! i guess it somehow got lost in the shuffle? do you want to (re)rename it or should i?

Reply:
I can do it!

Reply:
I did it on this same branch

Reply:
I fixed it above too (on line 29). Hence, now it says outdated.

Reply:
Thanks for fixing this link. What I meant is are you going to actually rename the file how-do-i-submit to how-do-i-register? because -register still 404s. http://obofoundry.org/faq/how-do-i-register-my-ontology.html

Reply:
I did rename the file, you can see it on the files changed above:
![image](https://user-images.githubusercontent.com/6722114/94644807-1f1d4680-029f-11eb-9ea3-de7b57592e5c.png)